packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-upgrade-version.command.ts

```diff
@@ -38,19 +38,19 @@ export class UpgradeTo0_40Command extends ActiveWorkspacesCommandRunner {
       'Running command to upgrade to 0.40: must start with phone calling code otherwise SyncMetadata will fail',
     );
 
-    await this.recordPositionBackfillCommand.executeActiveWorkspacesCommand(
+    await this.phoneCallingCodeCreateColumnCommand.executeActiveWorkspacesCommand(
```

Comment:
why changing the order?

Reply:
not intended.  the merge suggestion happened this way